flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -1103,7 +954,10 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
      * Removes Type II Message
      * @param message {string} Message to be removed
      */
-    tryRemoveMessage(message = this.inOut) {
+    tryRemoveMessage(message) {
```

Comment:
**issue/thought**: I think the `this.onClr = () => {` callback on line 92 might directly invoke this function, while it maybe should invoke the function which clears it on both sides. Not sure though.

Reply:
I haven't found a reference in a doc... but ToLiss and FF only clear the message on the onside MCDU (and ND if applicable)...

Reply:
how does the FSLabs does it? I can't really believe that each mcdu has it's own queue. Imagine flying with 3 mcdus, you'd be continuously clearing messages on it xD